eng/pipelines/client.yml

```diff
@@ -88,6 +89,18 @@ jobs:
           publishJUnitResults: false
           goals: 'install'
 
+      - task: Maven@3
+        displayName: 'Install azure-sdk-parent'
+        inputs:
+          mavenPomFile: parent/pom.xml
```

Comment:
Is this necessary because we have the same version of the parent pom in the repo as on maven but we require new changes that aren't on maven?

Reply:
It's not necessary but I have seen `Run SpotBugs` task failing because it couldn't find some dependency packages. There's no logical explanation to this as the previous `install` steps worked but the only diff in this pipeline and the PR pipeline is the `install parent pom` task. So, I added it to check if that makes a difference.

Reply:
@weshaggard I'd recommended that @srnagar try this. The spotbugs compilation failures were utterly baffling since the only thing different in the steps prior to it running was the installation of the parent pom.